addons/purchase_stock/models/product.py

```diff
@@ -32,6 +33,11 @@ def _get_buy_route(self):
 
     route_ids = fields.Many2many(default=lambda self: self._get_buy_route())
 
+    @api.depends('seller_ids')
+    def _compute_first_seller_id(self):
+        for template in self:
+            if len(template.seller_ids) > 0:
+                template.first_seller_id = template.seller_ids.sorted('sequence')[0].partner_id.id
```

Comment:
model use     _order = 'sequence, min_qty DESC, price, id'
no need for sorted here

Reply:
I tried it without `sorted` but didn't work correctly.